cpp/StatementPutGet.cpp

```diff
@@ -47,6 +71,17 @@ bool StatementPutGet::parsePutGetCommand(std::string *sql,
     putGetParseResponse->srcLocations.emplace_back(val->valuestring);
   }
 
+  cJSON* presignedUrls = (cJSON*)response->presigned_urls;
```

Comment:
I will check if presigned urls are still necessary with GCP communication

Reply:
Presigned urls could be skipped in the added code

Reply:
removed.